proto/storage/deployment.proto

```diff
@@ -19,15 +19,15 @@ message Deployment {
   reserved 3;
   uint64 hash = 26; // @gotags: hash:"ignore" sensorhash:"ignore"
   string type = 4; // @gotags: search:"Deployment Type"
-  string namespace = 5; // @gotags: search:"Namespace,store"
+  string namespace = 5; // @gotags: search:"Namespace,store" sql:"index=name:deployment_dashboard_sac;priority:3"
```

Comment:
Couple questions:
Did you check the resulting index in the database to make sure gorm applied the priority in the correct order?  It probably did, but worth a check.

Did you evaluate creating the indexes individually vs as a multi-column index?

Reply:
I thought risk score was already indexed, which was actually not the case.
Thanks for pointing out the single column option.
I benched 4.5.5 against a DB with a btree index on deployments.riskScore
| scope size | 4.5.5 | riskScore index |
|---|---|---|
| unrestricted | 32 ms | 9 ms |
| 1 | 27 ms | 26 ms |
| 10 | 29 ms | 29 ms |
| 20 | 30 ms | 30 ms |
| 50 | 33 ms | 33 ms |
| 100 | 38 ms | 34 ms |
| 200 | 50 ms | 39 ms |
| 500 | 177 ms | 47 ms |
| 1000 | 502 ms | 74 ms |
| 2000 | 1076 ms | 93 ms |

I'll just check GORM does create the index I expect here, but probably make the index here simpler (riskScore column only).

Reply:
Nice.  that is a significant improvement with just a minor change.